src/blosc2/lazyexpr.py

```diff
@@ -1933,6 +2020,31 @@ def lazyexpr(expression, operands=None, out=None, where=None):
     out: :ref:`LazyExpr`
         A :ref:`LazyExpr` is returned.
 
+    Examples
+    --------
+    >>> import blosc2
+    >>> import numpy as np
+    >>> dtype = np.float64
+    >>> shape = [3, 3]
+    >>> size = shape[0] * shape[1]
+    >>> a = np.linspace(0, 5, num=size, dtype=dtype).reshape(shape)
+    >>> b = np.linspace(0, 5, num=size, dtype=dtype).reshape(shape)
+    >>> a1 = blosc2.asarray(a)
+    >>> a1[:]
+    [[0.    0.625 1.25 ]
+    [1.875 2.5   3.125]
+    [3.75  4.375 5.   ]]
+    >>> b1 = blosc2.asarray(b)
+    >>> expr = 'a1 * b1 + 2'
+    >>> operands = { 'a': a1, 'b': b1 }
+    >>> lazy_expr = blosc2.lazyexpr(expr, operands=operands)
+    >>> f"Lazy expression created: {lazy_expr}"
+    Lazy expression created: a1 * b1 + 2
+    >>> expr_ = a1 * b1 + 2
+    >>> expr_[:]
+    [[ 2.        2.390625  3.5625  ]
+    [ 5.515625  8.25     11.765625]
+    [16.0625   21.140625 27.      ]]
```

Comment:
Maybe it would be better to show the results of the lazyexpr created with the blosc2.lazyexpr `lazy_expr` instead of creating the same expression and showing it (`expr_`)